### PR TITLE
Add java examples to jwt-verification.

### PR DESCRIPTION
**Pull request by ConzorKingKong**
## Summary of change
Adds Java example to JWT verification.

## Related issues
- https://github.com/supertokens/docs/issues/5 - Final checkbox. Only adds Java examples

### CODE REVIEW

File: v2/emailpassword/common-customizations/sessions/with-jwt/jwt-verification.mdx [hunk 1]
```diff
@@ -80,13 +80,22 @@ Refer to this [Github gist](https://gist.github.com/rishabhpoddar/8c26ed237add1a
   - This function takes care of caching public keys in memory + auto refetching if the public keys have changed (which happens automatically every 24 hours with supertokens). This will not cause any user logouts, and is just a security feature.
 - `main.go`: This is an example of how to verify a JWT using the golang JWT verification lib along with our helper function to get the JWKs keys. If you are using header based auth, you can fetch the JWT from the `Authorization Bearer` header, otherwise for cookie based auth, you can fetch it from the `sAccessToken` cookie.
 
+</TabItem>
+<TabItem value="java">
+
+Refer to this [Github gist](https://gist.github.com/ConzorKingKong/47e2723ede54079149f5f7c11e44c164) for a code reference of how use the Java `nimbus-jose-jwt` lib to do session verification. The gist contains three files:
+- `JWTVerification.java` You will have to modify the `coreUrl` in this file to point to your supertokens core instance (replacing the `try.supertokens.com` part of the URL).
```
Comment (rishabhpoddar) [inline v2/emailpassword/common-customizations/sessions/with-jwt/jwt-verification.mdx:87]: - JWTVerification.java -> no such file name in the gist. 
- no such variable called coreUrl

File: v2/emailpassword/common-customizations/sessions/with-jwt/jwt-verification.mdx [hunk 2]
```diff
@@ -80,13 +80,22 @@ Refer to this [Github gist](https://gist.github.com/rishabhpoddar/8c26ed237add1a
   - This function takes care of caching public keys in memory + auto refetching if the public keys have changed (which happens automatically every 24 hours with supertokens). This will not cause any user logouts, and is just a security feature.
 - `main.go`: This is an example of how to verify a JWT using the golang JWT verification lib along with our helper function to get the JWKs keys. If you are using header based auth, you can fetch the JWT from the `Authorization Bearer` header, otherwise for cookie based auth, you can fetch it from the `sAccessToken` cookie.
 
+</TabItem>
+<TabItem value="java">
+
+Refer to this [Github gist](https://gist.github.com/ConzorKingKong/47e2723ede54079149f5f7c11e44c164) for a code reference of how use the Java `nimbus-jose-jwt` lib to do session verification. The gist contains three files:
```
Comment (rishabhpoddar) [inline v2/emailpassword/common-customizations/sessions/with-jwt/jwt-verification.mdx:86]: In the gist, why does `JOSEException e` yield a 500 and not 401?

**Review by rishabhpoddar — CHANGES_REQUESTED** [review]

File: v2/emailpassword/common-customizations/sessions/with-jwt/jwt-verification.mdx (hunk 1 above)
Comment (ConzorKingKong) [reply v2/emailpassword/common-customizations/sessions/with-jwt/jwt-verification.mdx]: Gist updated to have `JWTVerification.java` and a CORE_URL variable

File: v2/emailpassword/common-customizations/sessions/with-jwt/jwt-verification.mdx (hunk 2 above)
Comment (ConzorKingKong) [reply v2/emailpassword/common-customizations/sessions/with-jwt/jwt-verification.mdx]: Changed to a 401

File: v2/emailpassword/common-customizations/sessions/with-jwt/jwt-verification.mdx [hunk 3]
```diff
@@ -80,13 +80,22 @@ Refer to this [Github gist](https://gist.github.com/rishabhpoddar/8c26ed237add1a
   - This function takes care of caching public keys in memory + auto refetching if the public keys have changed (which happens automatically every 24 hours with supertokens). This will not cause any user logouts, and is just a security feature.
 - `main.go`: This is an example of how to verify a JWT using the golang JWT verification lib along with our helper function to get the JWKs keys. If you are using header based auth, you can fetch the JWT from the `Authorization Bearer` header, otherwise for cookie based auth, you can fetch it from the `sAccessToken` cookie.
 
+</TabItem>
+<TabItem value="java">
+
```
[the remark this answers is not in the document]
Comment (ConzorKingKong) [reply v2/emailpassword/common-customizations/sessions/with-jwt/jwt-verification.mdx:85]: Okay. I updated the gist link and pushed up changes

**Pull request merged**